src/content/fr/2019/accessibility.md

```diff
@@ -0,0 +1,309 @@
+
+## Facilité de lecture
+
+Le principal objectif d’une page Web est de fournir du contenu avec lequel les utilisateurs souhaitent interagir. Ce contenu peut être une vidéo ou un assortiment d’images, mais souvent c’est simplement le texte sur la page. Il est extrêmement important que nos contenus textuels soient lisibles pour nos lecteurs. Si les visiteurs ne peuvent pas lire une page Web, ils ne peuvent pas interagir avec, ce qui se termine par un abandon de leur part. Dans cette section, nous examinerons trois domaines dans lesquels les sites ont connu des difficultés.
+
+### Contraste des couleurs
+
+Il existe de nombreux cas où les visiteurs de votre site peuvent ne pas le voir parfaitement. Les visiteurs peuvent être daltoniens et être dans l’impossibilité de faire la distinction entre la police et la couleur d’arrière-plan ([1 homme sur 12 et 1 femme sur 200](http://www.cvrl.org/people/stockman/pubs/1999%20Genetics%20chapter%20SSJN.pdf) en Europe). Peut-être qu’ils lisent simplement en extérieur avec un soleil créant des tonnes de reflets sur leur écran — ce qui nuit considérablement à leur vision. Ou peut-être qu’ils ont simplement vieilli et que leurs yeux ne peuvent pas distinguer les couleurs aussi bien qu’auparavant.
+
+Afin de vous assurer que votre site Web soit lisible dans ces conditions, un contraste de couleur suffisant entre le texte et son arrière-plan est capital.
+
+<figure>
+  <a href="/static/images/2019/accessibility/example-of-good-and-bad-color-contrast-lookzook.png">
+    <img alt="Figure 1. Exemple de texte présentant un contraste de couleurs insuffisant. Gracieusement mis à disposition par LookZook" aria-labelledby="fig1-caption" aria-describedby="fig1-description" src="/static/images/2019/accessibility/example-of-good-and-bad-color-contrast-lookzook.png" width="600" height="278">
+  </a>
+  <div id="fig1-description" class="visually-hidden">Quatre boîtes colorées de nuances orange et grises avec, par dessus, du texte blanc à l’intérieur créant deux colonnes. La colonne de gauche indique «&nbsp;Trop légèrement coloré&nbsp;» et a la couleur de fond orange écrite comme `#FCA469`. La colonne de droite indique «&nbsp;Recommandé&nbsp;» et la couleur d’arrière-plan orange est écrite comme `#F56905`. La zone supérieure de chaque colonne a un fond orange avec du texte blanc `#FFFFFF` et la zone inférieure a un fond gris avec du texte blanc `#FFFFFF`. Le fond gris montre la couleur orange en niveaux de gris. Gracieusement mis à disposition par LookZook</div>
+  <figcaption id="fig1-caption">Figure 1. Exemple de texte présentant un contraste de couleurs insuffisant. Gracieusement mis à disposition par LookZook</figcaption>
+</figure>
+
+Seuls 22,04% des sites ont donné à l’ensemble de leurs textes un contraste de couleurs suffisant. En d’autres termes&nbsp;: 4 sites sur 5 ont un texte qui se confond facilement avec son arrière-plan, le rendant illisible.
```

Comment:
Do we not have spaces before % marks in French as per comment  https://github.com/HTTPArchive/almanac.httparchive.org/issues/539#issue-527885745
> Microtypography (spaces, punctuation, apostrophes): if possible, apply basic rules (guillemets, spaces before ":" or "%") but not if it slows you down. We can always make a pass at the end to set everything up all at once (I know the regexes to apply).

```suggestion
Seuls 22,04 % des sites ont donné à l’ensemble de leurs textes un contraste de couleurs suffisant. En d’autres termes&nbsp;: 4 sites sur 5 ont un texte qui se confond facilement avec son arrière-plan, le rendant illisible.
```
A few instances of this but won't highlight the rest of these.

Reply:
I was hesitating to, I did not remember, will add them :)

Reply:
Done via https://github.com/HTTPArchive/almanac.httparchive.org/pull/759/commits/6034d35fdbe5a373b7f5529a8012e6e00e22ac67